### ISSUE TITLE
"0.12.0-next.8" `build` and `watch` have different output in "*.modern.js"

### ISSUE THREAD

**Issue by qwabra** [developit/microbundle#560]
```
$ yarn microbundle -v
microbundle, 0.12.0-next.8
```

```
$ microbundle build --target=web --compress=false
Build "qwa" to dist:
      1.53 kB: qwa.js.gz
      1.33 kB: qwa.js.br
      1.53 kB: qwa.modern.js.gz
      1.34 kB: qwa.modern.js.br
      1.53 kB: qwa.esm.js.gz
      1.34 kB: qwa.esm.js.br
      1.67 kB: qwa.umd.js.gz
      1.45 kB: qwa.umd.js.br
```

```
$ microbundle watch --target=web --compress=false
Watching source, compiling to dist:
Wrote 488 B: qwa.modern.js.gz
        408 B: qwa.modern.js.br
Wrote 1.67 kB: qwa.umd.js.gz
      1.45 kB: qwa.umd.js.br
Wrote 1.53 kB: qwa.esm.js.gz
      1.34 kB: qwa.esm.js.br
Wrote 1.53 kB: qwa.js.gz
      1.33 kB: qwa.js.br
```

**Comment by ForsakenHarmony** [comment]
can you share more info about your project?

possibly the output of the build/watch

**Comment by developit** [comment]
This is likely an issue with your `.babelrc` / `babel.config.js`. Can you share it?

We have to run through the babelrc to find where to inject preset-modules, and my guess is we are missing and it's getting an identical configuration to the non-modern output.

**Comment by qwabra** [comment]
https://github.com/qwabra/microbundle-bug

**Comment by developit** [comment]
This should be fixed by #570.
